databuilder/databuilder/task/mysql_staleness_removal_task.py

```diff
@@ -97,9 +97,20 @@ def _get_target_table_model_dict(self, target_tables: Set[str]) -> Dict[str, Typ
         :return:
         """
         target_table_model_dict: Dict[str, Type[RDSModel]] = {}
-        for model in Base._decl_class_registry.values():
+        print(f'{target_tables=}')
```

Comment:
please remove the print line before merge

Reply:
there are some issues with this change in 3.8 we are working through rn

Reply:
thanks, we will, just wanted to show Allison something